pkg/config/security/security.go

```diff
@@ -100,6 +105,36 @@ func CheckEmptyValues(key string, values []string) error {
 	return nil
 }
 
+func CheckValidPathTemplate(key string, paths []string) error {
```

Comment:
Moving this into pilot/pkg/security/authz/matcher/template.go will look more cohesive.